packages/samples/vuejswithcli/src/main.ts

```diff
@@ -0,0 +1,15 @@
+import {applyPolyfills, defineCustomElements} from '@coveo/atomic/loader';
+import '@coveo/atomic/dist/atomic/themes/coveo.css';
```

Comment:
This is nice to add as an example, since this will load the default theme.

An alternative is for people to define all the variables in `style.css`.

Reply:
nice, I guess it was coming automatically via the cdn!